src/lj_err.c

```diff
@@ -384,11 +385,15 @@ typedef struct UndocumentedDispatcherContext {
   ULONG ScopeIndex;
   ULONG Fill0;
 } UndocumentedDispatcherContext;
+#else
+typedef void* UndocumentedDispatcherContext;
+#endif
 
 /* Another wild guess. */
-extern void __DestructExceptionObject(EXCEPTION_RECORD *rec, int nothrow);
+extern void LJ_CDECL __DestructExceptionObject(EXCEPTION_RECORD *rec,
+					       int nothrow);
```

Comment:
Everything will break, if you build with said compiler flags. You'd have to tag more or less ALL C functions ...